__tests__/app.test.js

```diff
@@ -63,3 +62,42 @@ it('responds with 404 error if not path is found', ()=>{
     })
   })
 })
+
+describe.only('POST', () => {
+  test('post new comment to article and responds with the new comment posted', () => {
+
+    const newComments = {
+      username: "icellusedkars",
+      body: "nice work icellusedkars"
+    }
+    return request(app).post('/api/articles/4/comments').send(newComments).expect(201).then((response) => {
+      expect(response.body.comment).toMatchObject({
+        comment_id: 19,
+          author: "icellusedkars",
+          article_id: 4,
+          votes: 0,
+          body: "nice work icellusedkars"
+      })
+    })
+  })
+  test('responds with 404 when no input is given', ()=>{
+    const testOutput = {
+      username: '',
+      body: ''
+    }
+    return request(app).post('/api/articles/4/comments').send(testOutput).expect(404).then(({body: {msg}}) => {
+      expect(msg).toBe('value or type is not permitted!')
+
+    })
+  })
+  test('responds with 404 when input type is not permitted', ()=>{
+    const testOut = {
+      username: 13,
+      body: 34
+    }
+    return request(app).post('/api/articles/4/comments').send(testOut).expect(404).then(({body: {msg}})=> {
+      expect(msg).toBe(('value or type is not permitted!') 
+    )})
+})
```

Comment:
Status 404 Article does not exist  is missing - POST /api/articles/9999/comments